index.html

```diff
@@ -81,7 +81,6 @@ <h3>LET’S CONNECT</h3>
         </article>
       </section>
     </div>
-
     <section id="portfolio" title="portfolio-snapshot" class="card-works-box">
```

Comment:
- [ ] As asked by the previous reviewer you should make sure that  these section is generated with JavaScript.  Kindly remove the contents of this section and generate the data dynamically from the array of objects you have in the JavaScript file as suggested by the previous reviewer. Please check the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) as it is stated there. :handshake:

You did it for your pop-up, you should do the same for the projects!

1) Create an array of data, or you can also use the `projects` array you created, because they contain the same data.
2) Create another HTML template, as you did for the `creatPopup()`. You can create a `createProject()` function to do that